draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -11874,6 +11888,16 @@ Content-Encoding: gzip
   </front>
 </reference>
 
+<reference anchor="BREACH" target="http://breachattack.com/resources/BREACH%20-%20SSL,%20gone%20in%2030%20seconds.pdf">
+   <front>
+      <title>BREACH: Reviving the CRIME Attach</title>
+      <author initials="Y." surname="Gluck" fullname="Yoel Gluck"/>
+      <author initials="N." surname="Harris" fullname="Neal Harris"/>
+      <author initials="A." surname="Prado" fullname="Angelo Prado"/>
+      <date year="2013" month="July"/>
+   </front>
+</reference>
```

Comment:
CRIME might be better (or in addition to...), but I see that this is even sketchier as a reference (the breach paper cites a Google Slides presentation.

Reply:
Yeah, CRIME is problematic; at least this is a paper.